modules/gapi/test/serialization/core.hpp

```diff
@@ -0,0 +1,27 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_SRLZ_CORE_API_HPP
+#define OPENCV_GAPI_SRLZ_CORE_API_HPP
+
+#include <opencv2/gapi/gkernel.hpp> // GKernelPackage
+#include <opencv2/gapi/own/exports.hpp> // GAPI_EXPORTS
+
+namespace cv {
+namespace gapi {
+namespace core {
+namespace serialization {
```

Comment:
This header looks too public while it is actually not. What I recommend:

```suggestion
namespace opencv_test {
namespace serialization {
```

Reply:
Done